sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -33,28 +45,6 @@
 - Upgraded `azure-core` from `1.51.0` to version `1.52.0`.
 
 
-## 11.8.0-beta.1 (2024-09-23)
```

Comment:
Is this intended? remove this past changelog?

Reply:
No, I just ran the script this shouldn't have happened. I'll fix this.